constructor/header.sh

```diff
@@ -492,28 +492,42 @@ if [ "$BATCH" = "0" ]; then
        [ "$ans" != "y" ]   && [ "$ans" != "Y" ]
     then
         printf "\\n"
-        printf "You may wish to edit your .bashrc to prepend the __NAME__ install location to PATH:\\n"
+        printf "You may wish to edit your $BASH_RC to setup __NAME__:\\n"
         printf "\\n"
-        printf "export PATH=%s/bin:\$PATH\\n" "$PREFIX"
+        if [ -f "$PREFIX/etc/profile.d/conda.sh" ]; then
+            printf "source $PREFIX/etc/profile.d/conda.sh\\n"
+        else
+            printf "export PATH=\"$PREFIX/bin:\$PATH\"\\n"
+        fi
```

Comment:
I would make this part just

    printf "source $PREFIX/etc/profile.d/conda.sh\\n"

Not any of the other fallback code.

Reply:
Seems legit, since we are going to bump the minimum req on conda anyway.